extras/devices/__init__.py

```diff
@@ -1,6 +1,6 @@
 from devices.base import AbstractDevice, logger
-from devices.LVFS.LVFS import LVFSDevice, LenovoNB
-from devices.OEM.OEM import OEMDevice, HPEDevice
+from devices.LVFS import LVFSDevice, LenovoNB
+from devices.OEM import OEMDevice, HPEDevice
```

Comment:
You're right. I was intending to shorten the import string hierarchy but didn't think of the confusion it may cause. Thanks for the feedback!

Reply:
No, you don't have to remove all the __init__.py files. You can leave the one that makes it possible to write `from devices.LVFS import LVFSDevice` because writing `devices.LVFS.LVFS ` instead is indeed too long.

Reply:
You can just empty the `__init__` under the devices directory, but keep your changes under the ./LVFS and ./OEM dirs

Reply:
`devices/__init__.py` is actually widely-used as it facilitate the import of all classes with a single line `from devices import *`. So I prefer not to remove it. 
As for `__init__` under ./LVFS and ./OEM dirs, I created them just for shortening the import line in `devices/__init__.py`, in other words, it not being used other than in `devices/__init__.py`.

Reply:
Probably I miss something, but I don't see any init files in the diff. Probably you accidentally removed the init file under the devices dir

Reply:
There's already `devices/__init__.py`  in my previous PR so it doesn't show up in the diff.

Reply:
ah, I got it, thanks for pointing this out!